src/pages/MissionAndPurpose/MissionAndPurpose.js

```diff
@@ -18,7 +18,7 @@ const MissionAndPurpose = ({ data }) => {
   if (!data) return null;
 
   let { builder } = data;
-
+  
```

Comment:
Looks like this still needs to be addressed: https://github.com/Hawaian-United-Okinawa-Association/huoa-react/blob/5da0292d010e51ebf630f795d36986786a57fa91/src/pages/MissionAndPurpose/MissionAndPurpose.js#L1-L11

Reply:
If we could get that Title Body component implemented